router/pkg/metric/connection_measurements.go

```diff
@@ -0,0 +1,66 @@
+package metric
+
+import (
+	"fmt"
+	otelmetric "go.opentelemetry.io/otel/metric"
+)
+
+// Connection metric constants
+const (
+	maxConnections            = "router.http.client.max_connections"
+	connectionsActive         = "router.http.client.active_connections"
+	connectionAcquireDuration = "router.http.client.connections.acquire_duration"
```

Comment:
To follow the naming schema we could also use `router.http.client.acquire_connection_duration`

https://opentelemetry.io/docs/specs/semconv/http/http-metrics/#metric-httpclientopen_connections

Reply:
I thought we went aligned on prefixing it with `connections` because for future metrics like `router.http.client.connections.dns_duration` etc everything will be prefixed by connections are you suggesting that for future metrics we also don't prefix, i.e. :- router.http.client.dns_connection_duration ?